lib/Parser/RegexRuntime.h

```diff
@@ -453,17 +454,7 @@ namespace UnifiedRegex
 #endif
     };
 
-    struct HardFailMixin
-    {
-        bool canHardFail;
-
-        inline HardFailMixin(bool canHardFail) : canHardFail(canHardFail) {}
-
-#if ENABLE_REGEX_CONFIG_OPTIONS
-        void Print(DebugWriter* w, const char16* litbuf) const;
-#endif
-    };
-
+#pragma pack(push, 1)
```

Comment:
only member is an int, shouldn't this already be packed?

Reply:
I packed this one because the place it was included warn-as-error about the mixin being unaligned.

In the latest push every bytecode is packed.

Reply:
I had some redundant ones. Removed now.